elm/Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
```

Comment:
Do we actually need this file or is it an artifact of some env manager?

Reply:
Looks like it came from Francisco's patch.
@TheExGenesis Looks like some kind of docker stuff. Any ideas?